lib/travis/build/appliances/git_wire_protocol_v2.rb

```diff
@@ -0,0 +1,13 @@
+require 'travis/build/appliances/base'
+
+module Travis
+  module Build
+    module Appliances
+      class GitWireProtocolV2 < Base
+        def apply
```

Comment:
Could there be a clause specifying the images where this should apply?
Reason is mostly to have it documented where this should apply, so that this workaround can get removed when the images have been fixed/removed.

Reply:
Could that be something we add after?